migrations/2023-07-06-151124_hot-rank-future/up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE FUNCTION hot_rank(score numeric, published timestamp without time zone)
+    RETURNS integer
+    AS $$
+DECLARE
+    hours_diff numeric := EXTRACT(EPOCH FROM (timezone('utc', now()) - published)) / 3600;
+BEGIN
+    IF (hours_diff > 0) THEN
+        RETURN floor(10000 * log(greatest(1, score + 3)) / power((hours_diff + 2), 1.8))::integer;
```

Comment:
Rather than the if statement, it'd probable be better to do `greatest(0, hours_diff) + 2`

Reply:
Ah, makes sense.